Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN go get -u github.com/jstemmer/go-junit-report && \
     go get github.com/smartystreets/goconvey
 
 RUN go build -a -installsuffix cgo \
-    -ldflags="-X github.com/cyberark/conjur-authn-k8s-client/pkg/authenticator.Tag=$TAG" \
+    -ldflags="-X 'github.com/cyberark/conjur-authn-k8s-client/pkg/authenticator.TagSuffix=$TAG_SUFFIX'" \
```

Comment:
why did you add the surrounding `'`? do we need it?

Reply:
due to the space that comes up in a non-fixed versions..